### ISSUE TITLE
Modules were resolved with conflicting cross-version

### ISSUE THREAD

**Issue by ravwojdyla** [dataArtisans/flink-dataflow#19]
When depending on flink-dataflow:

```
error] Modules were resolved with conflicting cross-version suffixes in:
[error]    org.apache.flink:flink-clients _2.10, <none>
[error]    org.apache.flink:flink-runtime _2.10, <none>
[error]    org.apache.flink:flink-optimizer _2.10, <none>
[error]    org.apache.flink:flink-avro _2.10, <none>
```

Is there underlying flink version is `1.0-SNAPSHOT`
Any hint how to fix?

**Comment by mxm** [comment]
Yes, please switch to flink version 1.0.0 :)

**Comment by mxm** [comment]
I little bit more explanation: In the snapshot version, there was a bug which could pull in different Scala versions. This has been resolved for `1.0.0`. I'll close this for now, but feel free to re-open if this problems persists.

**Comment by ravwojdyla** [comment]
I rebased on current master (with flink version `1.0.0`) but the problem is still there - the problem __seems__ to be that dataflow-flink runner explicitly depends scala 2.10 artifacts:

```
[info] downloading https://repo1.maven.org/maven2/org/apache/flink/flink-streaming-java_2.10/1.0.0/flink-streaming-java_2.10-1.0.0.jar ...
[info]  [SUCCESSFUL ] org.apache.flink#flink-streaming-java_2.10;1.0.0!flink-streaming-java_2.10.jar (1343ms)
[info] downloading https://repo1.maven.org/maven2/org/apache/flink/flink-clients_2.10/1.0.0/flink-clients_2.10-1.0.0.jar ...
[info]  [SUCCESSFUL ] org.apache.flink#flink-clients_2.10;1.0.0!flink-clients_2.10.jar (1087ms)
[info] downloading https://repo1.maven.org/maven2/org/apache/flink/flink-connector-kafka-0.8_2.10/1.0.0/flink-connector-kafka-0.8_2.10-1.0.0.jar ...
[info]  [SUCCESSFUL ] org.apache.flink#flink-connector-kafka-0.8_2.10;1.0.0!flink-connector-kafka-0.8_2.10.jar (98ms)
[info] downloading https://repo1.maven.org/maven2/org/apache/flink/flink-avro_2.10/1.0.0/flink-avro_2.10-1.0.0.jar ...
[info]  [SUCCESSFUL ] org.apache.flink#flink-avro_2.10;1.0.0!flink-avro_2.10.jar (50ms)
[info] downloading https://repo1.maven.org/maven2/org/apache/flink/flink-connector-kafka-base_2.10/1.0.0/flink-connector-kafka-base_2.10-1.0.0.jar ...
[info]  [SUCCESSFUL ] org.apache.flink#flink-connector-kafka-base_2.10;1.0.0!flink-connector-kafka-base_2.10.jar (64ms)
[info] Done updating.
[error] Modules were resolved with conflicting cross-version suffixes in {file:/Users/rav/projects/scio/}scio-flink:
[error]    com.twitter:chill _2.11, _2.10
[trace] Stack trace suppressed: run last scio-flink/*:update for the full output.
[error] (scio-flink/*:update) Conflicting cross-version suffixes in: com.twitter:chill
[error] Total time: 86 s, completed Mar 10, 2016 11:15:21 AM
> scalaVersion
[info] 2.11.8
```

Maybe flink-dataflow should be released for both scala 2.10 and 2.11?

**Comment by mxm** [comment]
You brought in Chill 2.11 through some other dependency, right? In that case you have to use the `_2.11` suffixed Maven modules. We can probably just introduce a property in the `pom.xml` which you override when you want to use Scala 2.11.
